src/apps/intermediate-list/modal/my-payment-overview-modal.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { FC } from "react";
+import ExternalLinkIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/buttons/icon-btn-external-link.svg";
+import Modal, { useModalButtonHandler } from "../../../core/utils/modal";
+import { useText } from "../../../core/utils/text";
+
+const MyPaymentOverviewModal: FC = () => {
+  const t = useText();
+  const { close } = useModalButtonHandler();
+  const openInNewTab = (url: URL) => {
+    window.open(url, "_blank", "noopener,noreferrer");
+  };
+  const handleClickhehe = () => {
+    close("intermediate-payment-modal");
+  };
+  return (
+    <Modal
+      modalId="intermediate-payment-modal"
+      closeModalAriaLabelText="test"
+      screenReaderModalDescriptionText="test"
+      classNames="modal-cta"
+    >
+      <div className="modal-cta__container">
+        <h2 className="text-header-h2">
+          {t("intermediatePaymentModalHeaderText")}
+        </h2>
+        <div className="mt-48 color-secondary-gray">
+          <p className="text-body-medium-regular">
+            {t("intermediatePaymentModalBodyText")}
+          </p>
+          <p className="text-links mt-24">
+            {t("intermediatePaymentModalNoticeText")}
+          </p>
+        </div>
+        <div className="modal-cta__buttons mt-48">
+          <button
```

Comment:
https://github.com/danskernesdigitalebibliotek/dpl-react/blob/5016e2ed5ae78d4d13d98435bd3e7d0ff9dd6986/src/components/atoms/link.tsx#L4 use this and the classname props should be "btn-primary btn-filled btn-large arrow__hover--right-small"